.github/workflows/ci-rs.yml

```diff
@@ -221,6 +235,15 @@ jobs:
           flags: rust
           token: ${{ secrets.CODECOV_TOKEN }}
 
+  rs-semver-checks:
+    needs: changes
+    if: ${{ needs.changes.outputs.rust == 'true' && github.event_name == 'pull_request' }}
+    uses: CQCL/hugrverse-actions/.github/workflows/rs-semver-checks.yml@ab/semver-checks
```

Comment:
I'll change the target branch as soon as https://github.com/CQCL/hugrverse-actions/pull/13 is merged